swagger/src/test/resources/swagger.json

```diff
@@ -17,6 +17,7 @@
     "/store/order": {
       "post": {
         "operationId": "placeOrder",
+        "deprecated": false,
```

Comment:
swagger-ui should work without these `false` declarations. See [petstore swagger.json](http://petstore.swagger.io/v2/swagger.json) as an example.